integration/wai-aria.spec.js

```diff
@@ -39,240 +54,272 @@ describe('WAI ARIA Spec', () => {
             expect(title).toBe(
                 'React Accessible Accordion - Integration Test Sandbox',
             );
-
-            await browser.close();
         });
         it('has rendered the "classic accordion" example', async () => {
             const { browser, page } = await setup();
             const qtyClassicAccordion = await page.evaluate(
                 () => document.querySelectorAll('#classic-accordion').length,
             );
             expect(qtyClassicAccordion).toEqual(1);
-
-            await browser.close();
         });
     });
 
     describe('Chrome Accessibility Tree', () => {
         it('matches snapshots', async () => {
             const { browser, page } = await setup();
             expect(await page.accessibility.snapshot()).toMatchSnapshot();
-
-            await browser.close();
         });
     });
 
     describe('Keyboard Interaction', () => {
         describe('Enter or Space', () => {
-            it('When focus is on the accordion header for a collapsed panel, expands the associated panel. If the implementation allows only one panel to be expanded, and if another panel is expanded, collapses that panel.', async () => {
-                const { browser, page, headingsHandles } = await setup();
-                expect(headingsHandles.length).toEqual(3);
+            it(`When focus is on the accordion header for a collapsed panel,
+                expands the associated panel. If the implementation allows only
+                one panel to be expanded, and if another panel is expanded,
+                collapses that panel.`, async () => {
+                const { browser, page, buttonsHandles } = await setup();
+                expect(buttonsHandles.length).toEqual(3);
 
-                const firstHeadingHandle = headingsHandles[0];
-                const secondHeadingHandle = headingsHandles[1];
+                const firstButtonHandle = buttonsHandles[0];
```

Comment:
Using buttons now in most places instead of headings, because that's what the aria-spec actually calls for.